hermes/src/store.rs

```diff
@@ -341,7 +341,8 @@ mod test {
     /// Generate list of updates for the given list of messages at a given slot with given sequence
     ///
     /// Sequence in Vaas is used to filter duplicate messages (as by wormhole design there is only
-    /// one message per sequence)
+    /// one message per sequence).
+    #[cfg(test)]
```

Comment:
isn't the entire test module under this cfg?

Reply:
Ah I missed that, will remove.